core/player/src/view/resolver/index.ts

```diff
@@ -376,6 +376,21 @@ export class Resolver {
         childTreeDeps.forEach((binding) => childDependencies.add(binding));
 
         if (childValue) {
+          if (childNode.type === NodeType.Template) {
+            if (childNode.placement === "append") {
+              const arr = addLast(
+                dlv(resolved, child.path as any[], []),
+                childValue,
+              );
+              resolved = setIn(resolved, child.path, arr);
+            } else if (childNode.placement === "prepend") {
+              const arr = addFirst(
+                dlv(resolved, child.path as any[], []),
+                childValue,
+              );
+              resolved = setIn(resolved, child.path, arr);
+            }
+          }
```

Comment:
Do we still need this if we're sorting the dynamic template in the parser?

Reply:
TBH I forgot this was a thing - I just reverted these changes and everything still passes 👍